pkg/spec/controller.go

```diff
@@ -96,5 +99,44 @@ func CoreOperations(data []byte) ([]runtime.Object, []string, error) {
 		return nil, nil, errors.Wrap(err, "unable to transform data")
 	}
 
+	//Regex to find the location of appversion
```

Comment:
If I understand it correctly this is basically manually parsing appversion from YAML. This is not nice.
Please leave it as it was before.  It is better to set labels on multiple places than parsing it here like this.

Reply:
@kadel I did it because after that user can give input without quotes and also trailing zeroes and all the problem get solved.
I tried IntOrFloarOrString there I was facing trailing zeroes in float problem, otherwise we can use string and user have to give input in quotes